homeassistant/components/switch_as_x/config_flow.py

```diff
@@ -28,7 +28,7 @@
 class SwitchLightConfigFlowHandler(
     helper_config_entry_flow.HelperConfigFlowHandler, domain=DOMAIN
 ):
-    """Handle a config or options flow for Switch Light."""
+    """Handle a config flow for Switch as X."""
 
     steps = STEPS
 
```

Comment:
Store the platform that you want to set up the entry for in the entry data.

Reply:
Fixed 👍